services/serviceenablement/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## v0.1.0 (2024-05-17)
+
+- **New**: Manage STACKIT Service Enablement resources: `Services`
```

Comment:
We can make this more clear, by clearly stating that this module can be used to enable services, it's not really "managing resources"